src/pages/gen2/reference/amplifyconfiguration/index.mdx

```diff
@@ -1,3 +1,5 @@
+import { SubCommandHeading } from '@/components/CliCommands/CommandHeading';
```

Comment:
is the intent for using this to get anchors in the TOC?

Reply:
yes, would it be better if i just use `##`